stablehlo/transforms/Passes.td

```diff
@@ -258,3 +258,34 @@ def StablehloLegalizeQuantizedOpToQDQPass : Pass<"stablehlo-legalize-quantized-o
     "mlir::stablehlo::StablehloDialect",
   ];
 }
+
+def StablehloLegalizeQDQToQuantizedOpPass : Pass<"stablehlo-legalize-qdq-to-quantized-op", "mlir::func::FuncOp"> {
+  let summary = "Compose StableHLO quantized op using uniform quantize/dequantize ops";
```

Comment:
This summary does not seem to imply the reverse of "Decompose StableHLO quantized ops using uniform quantize/dequantize ops." (in the pass above)

How about:
"Compose uniform quantize/dequantize ops into StableHLO quantized ops"

Reply:
it does imply? `compose (put together)  StableHLO quantized op using uniform quantize/dequantize ops` ?

Reply:
This is what I am thinking: 

In the context of `StableHLOLegalizeQuantOpToQDQ`, the line `...using uniform quantize/dequantize` suggests that the destination artifacts. That is we decompose quant-op `using` q/dq op (the destination). 

Ideally, in the context of `StablehloLegalizeQDQToQuantizedOpPass`  the description should suggest that the dqd are in the source.  

Another option I can think of:  
quant-op -> qdq: Decompose quantized StableHLO operation to (de-quantize, floating-point operation and quantize) pattern. 
qdq->quant-op: Compose  (de-quantize, floating-point operation and quantize) pattern to StableHLO quantized operation.

This is just a minor suggestion for better readability (and hence subjective). Feel free to resolve.

Reply:
ack, thank you.
I am onboard with 
`Decompose quantized StableHLO operation to (de-quantize, floating-point operation and quantize) pattern.`
and 
`Fuse (de-quantize, floating-point operation and quantize) pattern into StableHLO quantized operation.`

will update it for both the passes.